arch/cortex-m4/build.rs

```diff
@@ -0,0 +1,16 @@
+extern crate gcc;
```

Comment:
No no no... we absolutely _do not_ want to put the crypto related assembly files in these crates. These crates are _minimal_ architecture specific code for the Tock kernel, and the crypto code is not even integrated anywhere into the kernel, let alone critical for it.